app/helpers/listings_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ListingsHelper
+  def listing_params
+    @listing_params = params.require(:listing).permit(:day, :hour, :mins)
+    @listing_params[:start_time] = params[:listing][:hour].to_i*60 + params[:listing][:mins].to_i
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)